go/runtime/bundle/manager.go

```diff
@@ -242,13 +257,38 @@ func (m *Manager) Queue(runtimeID common.Namespace, manifestHashes []hash.Hash)
 	}
 }
 
+// QueueCleanup updates the active version for the runtime ID,
+// triggering clean-up of regular bundles for version lower than active.
+func (m *Manager) QueueCleanup(runtimeID common.Namespace, active version.Version) {
+	// Update the active versions.
+	m.mu.Lock()
+	defer m.mu.Unlock()
+
+	m.cleanupQueue[runtimeID] = active
+
+	// Trigger immediate clean-up of stale bundles.
+	select {
+	case m.cleanupCh <- struct{}{}:
+	default:
+	}
+}
+
 // Download tries to download bundles in the queue.
 func (m *Manager) Download() {
 	for runtimeID := range m.runtimeIDs {
 		m.downloadBundles(runtimeID)
 	}
 }
 
+// CleanStaleBundles removes outdated manifest hashes and deletes corresponding
+// exploded bundles for runtimes in the clean-up queue.
+func (m *Manager) CleanStaleBundles() {
+	m.logger.Info("removing regular bundles with version less than active")
```

Comment:
```suggestion
	m.logger.Info("cleaning bundles")
```
If you are mentioning active version, one would like to know what the active version is. The better way is to just make a simple log, so that we know that the cleanup is triggered. Could also add a similar message `downloading bundles` to `Download`.